Providers/ProfilePrefs.swift

```diff
@@ -87,7 +87,11 @@ public class NSUserDefaultsProfilePrefs: Prefs {
     }
 
     public func stringArrayForKey(defaultName: String) -> [String]? {
-        return userDefaults.stringArrayForKey(qualifyKey(defaultName)) as [String]?
+        let objects = userDefaults.stringArrayForKey(qualifyKey(defaultName))
+        if let strings = objects as? [String] {
+            return strings
+        }
+        return nil
```

Comment:
Can this not be simplified to `return userDefaults.stringArrayForKey(qualifyKey(defaultName)) as? [String]`?